whois-api/src/main/java/net/ripe/db/whois/api/rdap/WhoisRdapService.java

```diff
@@ -152,7 +153,7 @@ public Response lookup(@Context final HttpServletRequest request,
                 return lookupObject(request, whoisObjectTypes, key);
             }
             case NAMESERVER: {
-                throw new NotFoundException("nameserver not found");
+                throw new ServerErrorException("nameserver search not implemented", Response.Status.NOT_IMPLEMENTED);
```

Comment:
Remove word 'search' - this is a lookup for a single nameserver. Also replace the word 'implemented' with 'supported'. (So: "Nameserver not supported").